packages/circuit-ui/components/Select/Select.tsx

```diff
@@ -390,7 +391,7 @@ export const Select = React.forwardRef(
             {...props}
             onChange={handleChange}
```

Comment:
The `defaultValue` needs to be manually forwarded to the `SelectElement` now (previously, it was spread as part of the remaining `props`).

Reply:
Oh, d'uh. I'll also check why the tests didn't catch that.